src/main/java/hudson/plugins/jira/UpdaterIssueSelector.java

```diff
@@ -27,6 +27,6 @@
      * @param listener Current's run listener.
      * @return Set of ids of issues which should be updated.
      */
-    public abstract Set<String> findIssueIds(@Nonnull Run<?, ?> run, @Nonnull JiraSite site, @Nonnull TaskListener listener);
+    public abstract Set<String> findIssueIds(@Nonnull Run<?, ?> run, @Nonnull Updater updater, @Nonnull JiraSite site, @Nonnull TaskListener listener);
```

Comment:
Second time somebody mentions API break. 
@jan-zajic can you preserve the old API with @deprecated?